pubspec.yaml

```diff
@@ -14,26 +14,28 @@ dependencies:
 
   # UI
   custom_switch_button: ^0.5.0
-  material_design_icons_flutter: ^4.0.5855
+  material_design_icons_flutter: ^4.0.5955
 
-  google_fonts: ^1.1.1
+  google_fonts: ^2.0.0
   flutter_svg: ^0.19.3
   md2_tab_indicator: ^1.0.2
 
-  firebase_core: ^1.0.0
-  firebase_analytics: ^7.1.0
+  firebase_core: ^1.0.1
+  firebase_analytics: ^7.1.1
   firebase_crashlytics: ^1.0.0
 
-  dio: ^3.0.10
-  flutter_bloc: ^6.1.3
-  shared_preferences: ^2.0.3
+  dio: ^4.0.0-prev3
```

Comment:
Let's use only stable versions of the libs.
Using preview version is not the best idea